### DISCUSSION TITLE
Format documentation for proxymansessionv2 files

### DISCUSSION THREAD

**Discussion by wabiloo** [ProxymanApp/Proxyman#2008] [Q&A]
Hi,

Is there any documentation anywhere I could find that describes the proxymansessionv2 files and how to use them?
I have a need to run an external process on captured sessions. I could export to .HAR first, but I think that the content of the proxymansessionv2 file is vastly superior, so I'd rather work with that if possible.

Cheers

**Comment by NghiaTranUIT** [comment]
No official document yet, but you can do it:
1. Rename to zip -> export it
2. It's a bunch of JSON files
